packages/vue/src/components/molecules/SfModal/SfModal.html

```diff
@@ -0,0 +1,16 @@
+<section class="sf-modal" :class="{ 'sf-modal--visible': visible }">
+  <transition :name="transitionOverlay">
+    <div class="sf-modal__overlay" v-show="visible" v-if="overlay" @click="checkPersistence"></div>
+  </transition>
+  <transition :name="transitionModal">
+    <div class="sf-modal__container" v-show="visible">
+      <button class="sf-modal__cross" @click="$emit('close')" v-if="cross">
+        <img src="/assets/close.svg" class="sf-circle-icon__icon" alt="">
```

Comment:
imo would be good to have a named slot here

Reply:
u mean in place of the img @leomp12 ?

Reply:
i mean containing the img as default content:

```vue
<slot name="close">
  <img src="/assets/close.svg" class="sf-circle-icon__icon" alt="">
</slot>
```